Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:16
+WORKDIR /var/www
+
+
+COPY ./package.json package.json
+RUN npm i
+COPY . .
+
+RUN ["npm", "run", "build"]
```

а какая фишка в массиве? Можно же просто написать

```suggestion
RUN npm run build
```